gateway-ha/src/test/java/io/trino/gateway/ha/TestGatewayHaMultipleBackend.java

```diff
@@ -136,7 +139,7 @@ public void testQueryDeliveryToMultipleRoutingGroups()
                         .post(requestBody)
                         .build();
         Response response1 = httpClient.newCall(request1).execute();
-        assertThat(response1.body().string()).contains("http://localhost:" + adhocTrino.getMappedPort(8080));
+        assertThat(response1.body().string()).contains("http://localhost:" + routerPort);
```

Comment:
This is needed because setting `http-server.process-forwarded` causes the `nextUrl` returns the URL of Trino Gateway, not the URL of Trino.